From: Jakub Jirutka <jakub@jirutka.cz>
Date: Sun, 16 Nov 2025 23:41:21 +0100
Subject: [PATCH] Change install umask to 0022

We run Kea as unprivileged user, so installing executables and
libraries with mode 750 is not desirable.

diff --git a/meson.build b/meson.build
index 4f99748..c7eba2b 100644
--- a/meson.build
+++ b/meson.build
@@ -26,7 +26,7 @@ project(
         'b_lundef=false',  # required for clang++ when used with sanitizers and for BSD ld when linking with extern char **environ.
         'b_pch=false',
         'b_pie=true',
-        'install_umask=0027',
+        'install_umask=0022',
         'warning_level=2',
     ],
 )
